include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -560,14 +560,19 @@ def TT_DotOp : TT_Op<"dot", [Pure,
     let summary = "dot";
 
     let description = [{
-        $d = matrix_multiply($a, $b) + $c
+        $d = matrix_multiply($a, $b) + $c. $f32Backend describes how to exercise the TC
+        when the inputs are f32. It can be one of: tf32, tf32x3, ieee.
+        tf32: use TC with tf32 ops.
+        tf32x3: implement the 3xTF32 trick. For more info see the pass in F32DotTC.cpp
+        ieee: don't use TC, implement dot in software.
+        If the GPU does not have Tensor cores or the inputs are not f32, this flag is ignored.
```

Comment:
Presumably these descriptions should be on the enum now.

Reply:
I think this comment still applies?

Reply:
I figured it'd be better to leave them in the place where one would first look at when finding what this flag really does, but sure, I can move them

Reply:
For example the enum is also used on dot_async.

(If you wanted a "see X" comment I'd be fine with that, although it's probably pretty obvious where to look.)